devolving_music/lib/song_scores.py

```diff
@@ -1,5 +1,6 @@
 from cmath import inf
 from typing import List
+from typing import Tuple
```

Comment:
Nit: from Python 3.9 onward, you can use the normal lowercase type names that are already in scope (e.g., `tuple`, `list`, `dict`) instead of importing the capitalized names from `typing`.

Reply:
fixed